### PR TITLE
idr034 fixes

### PR DESCRIPTION
**Pull request by joshmoore**
A few minor last minute adjustments

### CODE REVIEW

**Comment by manics** [comment]
Since you've added a test script how about adding travis?

File: submission.html [hunk 1]
```diff
@@ -96,7 +96,7 @@ <h1 class="hero-main-header">Submission of datasets to the IDR</h1>
                         <ol>
                             <li>Templates for <b>high content screens</b> (HCS),
                             or imaging studies performed in plates (e.g. 96 or 384 well plates)</li>
-                            <li>Templates for non-screen datasets, or studies which group images
+                            <li>Templates for <b>non-screen</b> datasets, or studies which group images
```
Comment (eleanorwilliams) [inline submission.html]: Change to  <b>non-screen studies</b>

**Comment by manics** [comment]
docker travis looks good to me

**Comment by joshmoore** [comment]
Cool. Merging. This will get tagged as IDR-0.3.4 then.

**Pull request merged**